packages/react-charting/src/components/PieChart/Arc/Arc.styles.ts

```diff
@@ -1,8 +1,11 @@
 import { IArcProps, IArcStyles } from './Arc.types';
-import { DefaultPalette } from '@fluentui/react/lib/Styling';
-export const getStyles = (props: IArcProps): IArcStyles => {
+import { DefaultPalette, ITheme } from '@fluentui/react/lib/Styling';
+export const getStyles = (props: IArcProps, theme: ITheme | undefined): IArcStyles => {
   const { color } = props;
   return {
-    root: { fill: color, stroke: DefaultPalette.white, strokeWidth: 2 },
+    pieRoot: { fill: color, stroke: theme?.palette.white || DefaultPalette.white, strokeWidth: 0 },
+    pieRootFocused: { fill: color, stroke: theme?.palette.black || DefaultPalette.black, strokeWidth: 3 },
```

Comment:
Did you get the focused strokeWidth = 3 from figma.

Reply:
I'll make it strokeWidth=2 to have consistency with other charts. Couldn't find in figma, will check again if something's mentioned regarding this